oak/server/rust/oak_abi/Cargo.toml

```diff
@@ -5,14 +5,9 @@ authors = ["David Drysdale <drysdale@google.com>"]
 edition = "2018"
 license = "Apache-2.0"
 
-[features]
-std = ["protobuf"]
-no_std = []
-default = ["std"]
-
 [dependencies]
-protobuf = { version = "*", optional = true }
+prost = "0.6"
```

Comment:
Could we use "*" here so it is more likely that all our crates will always depend on the same version?